### PR TITLE
bz2066992 updated installation with custom SCC

### PR DESCRIPTION
**Pull request by sheriff-rh**
CP to 4.7+
I put an IMPORTANT note at the bottom of each type of installation method:
[deploy-preview-44210--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-installation.html](https://deploy-preview-44210--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-installation.html)

In the note, I linked to the custom SCC procedure, which I placed in the advanced CO tasks section:
[deploy-preview-44210--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-advanced.html#compliance-custom-scc_compliance-advanced](https://deploy-preview-44210--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-advanced.html#compliance-custom-scc_compliance-advanced)

Per [bz2066992](https://bugzilla.redhat.com/show_bug.cgi?id=2066992):

* Added statements in the install sections that included custom SCC creation
* Added a procedure module to the Advanced Compliance Operator section, personally tested validation in cluster-bot; added additional resources for more info on SCCs
* Fixed a typo I found in another file while researching SCCs

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@Vincent056  PTAL! 

I put an `IMPORTANT` note at the bottom of each type of installation method: 
https://deploy-preview-44210--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-installation.html

In the note, I linked to the custom SCC procedure, which I placed in the advanced CO tasks section: 
https://deploy-preview-44210--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-advanced.html#compliance-custom-scc_compliance-advanced 

I also moved the `-MKNOD` in your yaml example (provided in the bug) under the `requiredDropCapabilities`, I think that's where you meant for that to go. `oc` wouldn't accept it otherwise.

Let me know what you think!

**Comment by Vincent056** [comment]
I think that looks really nice!

**Review by Vincent056 — APPROVED** [review]

**Comment by pdhamdhe** [comment]
LGTM
@xiaojiey Hi, please help to review. Thanks

File: modules/compliance-custom-scc.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-custom-scc_{context}"]
+= Creating a custom SCC for the Compliance Operator
+
+In some environments, creating a custom Security Context Constraints (SCC) file must be created to ensure the correct permissions are available to the Compliance Operator `api-resource-collector`.
```
Comment (EricPonvelle) [inline modules/compliance-custom-scc.adoc:9]: "be created" is passive voice, so maybe:

In some environments, you must create a custom Security Context Constraints (SCC) file...

**Review by EricPonvelle — COMMENTED** [review]
Small catch on the passive voice, but this looks good otherwise.

**Pull request merged**